docker/Dockerfile.umn

```diff
@@ -18,8 +18,16 @@ RUN apt-get update  && apt-get install -y \
     ros-humble-grid-map-rviz-plugin \
     ros-humble-can-msgs \
     ros-humble-usb-cam \
+    python3-pytest \
+    python3-opencv \
```

Comment:
Should this be moved to real sense layer or smth

Reply:
I think we need opencv anyways for apriltags and maybe also nvblox, i dont think it is only for realsense